### PR TITLE
1703 Side nav bar design

### PR DESCRIPTION
**Pull request by tiwarineelu59**

### CODE REVIEW

File: packages/tailwind-constructor/tailwind.config.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ module.exports = withTV({
       transparent: 'transparent',
       ...filterTokensByType('color', tokens),
     },
+    extend: {
+      colors: {
+        'sky-blue': '#7dd3fc',
+      }},
```
Comment (Enjoy2Live) [inline packages/tailwind-constructor/tailwind.config.js:30]: @tiwarineelu59 why are you adding this new color? this needs to be approved by design team.

@Nouri-Anouar please check if this should be added or not

**Review by Enjoy2Live — CHANGES_REQUESTED** [review]

File: packages/tailwind-constructor/tailwind.config.js (hunk 1 above)
Comment (tiwarineelu59) [reply packages/tailwind-constructor/tailwind.config.js:30]: @Enjoy2Live This color was used in the user profile design layout for one of the texts but was not able to apply so I had to add this in tailwind.confi.js

Comment (Nouri-Anouar) [reply packages/tailwind-constructor/tailwind.config.js:30]: Hello @tiwarineelu59 
Could you please provide me the Figma link where this color is being used?

Comment (Enjoy2Live) [reply packages/tailwind-constructor/tailwind.config.js:30]: @tiwarineelu59 just making sure you are receiving these notifications, if you do sorry for the additional ping!

Comment (tiwarineelu59) [reply packages/tailwind-constructor/tailwind.config.js:30]: @Nouri-Anouar https://www.figma.com/design/D31z19f4we75VkmEGzdyUn/User-Profile-%2F-Onboarding-Main-File-(Copy)?node-id=2239-18693&m=dev

**Review by dbradham — APPROVED** [review]
Changes in IdeaSpace are all related to cleaning code & refactoring. Looks good to me!

**Pull request closed without merging**